plugins/woocommerce-admin/client/core-profiler/components/progress-bar/progress-bar.tsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+
+import { HTMLAttributes } from 'react';
+/**
+ * Internal dependencies
+ */
+
+import './progress-bar.scss';
+import WooLogo from '../navigation/woologo';
```

Comment:
@moon0326 👋 It seems that linting is failing because of this unused import.

```
plugins/woocommerce-admin lint: /home/runner/work/woocommerce/woocommerce/plugins/woocommerce-admin/client/core-profiler/components/progress-bar/progress-bar.tsx
plugins/woocommerce-admin lint:   11:8  error  'WooLogo' is defined but never used. Allowed unused vars must match /createElement/u  @typescript-eslint/no-unused-vars
```

Reply:
Updated in https://github.com/woocommerce/woocommerce/pull/37741/commits/33598f10d677b084a12f82f915335f725c00d8b7